ci/check-abi.sh

```diff
@@ -31,14 +31,15 @@ if [ "${CHECK_ABI:-}" != "yes" ]; then
 fi
 
 exit_status=0
-for library in google_cloud_cpp_common bigtable_client; do
+for library in storage_client google_cloud_cpp_common bigtable_client; do
```

Comment:
Q. Is there an ordering we might prefer (e.g., google_cloud_cpp_common first)?

Reply:
I cannot think of any order that is much better than any other, so I am going with alphabetical for now.  We can change it if we learn this is not ideal.